operations/research-and-development/product/development-process/new-bug-tickets/bug-severity-guidelines.md

```diff
@@ -3,13 +3,13 @@
 ## Criteria for bugs
 
 * “Scale” of impact
-  * S1 - `Data loss or crash` - Prevents app use (e.g. Mattermost crashes when posting a message). These are the only bugs that can be committed after the T-5 Code Freeze deadline.
-  * S2 - `Functionality loss or glaring UI regressions` - Most users’ functionality affected (e.g. cannot favorite a channel) or includes functional and UI regressions that affect functionality. In addition to S1 bugs, these are the only bugs that can be committed after Code Complete deadline until T-5 Code Freeze.
```

Comment:
Are these bugs not longer accepted as cherry pick after code complete deadline?

Reply:
I've added back that sentence as it makes sense to keep it here.